gc/base/SlotObject.hpp

```diff
@@ -166,7 +166,7 @@ class GC_SlotObject
 #if defined (OMR_GC_COMPRESSED_POINTERS)
 	, _compressedPointersShift(((OMR_VM *)omrVM)->_compressedPointersShift)
 #if defined (OMR_GC_FULL_POINTERS)
-	, _compressObjectReferences(OMRVM_COMPRESS_OBJECT_REFERENCES(omrVM))
+	, _compressObjectReferences(OMRVM_COMPRESS_OBJECT_REFERENCES((OMR_VM *)omrVM))
```

Comment:
Do we use this constructor anywhere? I did brief search and can not find any callers. And if I missed them can we do cast on callers side instead of have this stupid constructor ?

Reply:
I'll check and remove it if it's unused.

Reply:
Done.